### ISSUE TITLE
Add method to retrieve a route's handler

### ISSUE THREAD

**Issue by cpsievert** [thomasp85/routr#7]
### Motivation

It can be useful to overwrite a route's handler by first obtaining that handler, then writing a new handler which may call the old handler. For an example, [Dash does this](https://github.com/plotly/dash-auth/blob/7a412877f1d1690ca1ac28f6048927e6c4fc565f/dash_auth/auth.py#L16) to enable an authenticated "superclass" via flask's view_functions property

### Implementation

Looking at the [current `remove_handler()` method](https://github.com/thomasp85/routr/blob/f1687a492fdac8564b7fcf288e437f7f275e9370/R/route.R#L158-L167), perhaps we could add a method that essentially does this?

```r
get_handler = function (method, path) {
    id <- private$find_id(method, path)
    if (is.null(id)) {
        warning("No handler assigned to ", method, " and ", path, 
            call. = FALSE)
    }
  get(id, envir = private$handlerStore)
}
```

If this sounds useful, I'll take a stab at a PR

**Comment by cpsievert** [comment]
Sorry I mistakenly posted this comment b4 it was ready. Will update...

**Comment by thomasp85** [comment]
Closed by #9
